pkg/kubelet/certificate/bootstrap/bootstrap_test.go

```diff
@@ -387,7 +387,7 @@ func TestRequestNodeCertificate(t *testing.T) {
 type failureType int
 
 const (
-	noError failureType = iota
+	noError failureType = iota //nolint
```

Comment:
What is this warning?

Reply:
I don't get a warning here when I run your PR manually?

Reply:
See this...

```
[dims@dims-a02 08:52] ~/go/src/k8s.io/kubernetes ⟩ golangci-lint --version
golangci-lint has version v1.41.1 built from (unknown, mod sum: "h1:KH28pTSqRu6DTXIAANl1sPXNCmqg4VEH21z6G9Wj4SM=") on (unknown)
[dims@dims-a02 08:52] ~/go/src/k8s.io/kubernetes ⟩ golangci-lint run --disable-all -E deadcode pkg/kubelet/certificate/bootstrap/
pkg/kubelet/certificate/bootstrap/bootstrap_test.go:390:2: `noError` is unused (deadcode)
	noError failureType = iota
	^
[dims@dims-a02 08:52] ~/go/src/k8s.io/kubernetes ⟩ golangci-lint run --disable-all -E varcheck pkg/kubelet/certificate/bootstrap/
pkg/kubelet/certificate/bootstrap/bootstrap_test.go:390:2: `noError` is unused (varcheck)
	noError failureType = iota
	^
```